src/System.Console/tests/NegativeTesting.cs

```diff
@@ -8,93 +8,54 @@
 
 public class NegativeTesting
 {
-    [Fact]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
-    public static void OpenStandardOutNegativeTests()
-    {
-        NegativeConsoleOutputTests(Console.OpenStandardOutput());
-    }
-
-    [Fact]
-    [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
+    [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsWinRT))] // In appcontainer, the stream cannot be opened: there is no Console
```

Comment:
We added a new TFM yesterday. TargetFrameworkMonikers.UapNonUapAot which is uap only. So instead of using conditional fact to make it more clear you could do `[SkipOnTargetFramework(~TargetFrameworkMonikers.UapNonUapAot)]`

See: https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/TargetFrameworkMonikers.cs#L25